anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
Ok. I think comments would be welcome, Anki is already hard enough to understand. (mind you, I'm just a passer-by...)